<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editUSER</class>
 <widget class="QDialog" name="editUSER">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thông tin người dùng</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/users.png</normaloff>:/images/users.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="7,2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Vai trò</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CMND/MSSV</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>24</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>32</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Họ tên</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>60</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nghề nghiệp</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>46</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Tài khoản</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>47</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ngày sinh</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_taikhoan">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_hoten">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cmnd">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ngaysinh">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>140</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nghenghiep">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Giới tính</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_gioitinh">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>230</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>170</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox_dg">
           <property name="text">
            <string>Độc giả</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_tt">
           <property name="text">
            <string>Thủ thư</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_ql">
           <property name="text">
            <string>Quản lý</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <zorder>lineEdit_cmnd</zorder>
       <zorder>lineEdit_email</zorder>
       <zorder>lineEdit_ngaysinh</zorder>
       <zorder>lineEdit_nghenghiep</zorder>
       <zorder>lineEdit_taikhoan</zorder>
       <zorder>lineEdit_hoten</zorder>
       <zorder>label_6</zorder>
       <zorder>label_3</zorder>
       <zorder>label_4</zorder>
       <zorder>label_2</zorder>
       <zorder>label_7</zorder>
       <zorder>label</zorder>
       <zorder>label_5</zorder>
       <zorder>checkBox_dg</zorder>
       <zorder>checkBox_tt</zorder>
       <zorder>checkBox_ql</zorder>
       <zorder>label_8</zorder>
       <zorder>lineEdit_gioitinh</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_pass">
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="lineEdit_mkmoi">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_xacnhan">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>31</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Mới</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Xác nhận</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Mật khẩu cũ</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mkcu">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1,1">
     <item>
      <widget class="QGroupBox" name="groupBox_user">
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_update">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Cập nhật</string>
        </property>
        <property name="statusTip">
         <string>Cập nhật thông tin đã thay đổi</string>
        </property>
        <property name="styleSheet">
         <string notr="true"> </string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/update2.png</normaloff>:/images/update2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Sửa thông tin</string>
        </property>
        <property name="statusTip">
         <string>Sửa đổi thông tin tài khoản này</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/ucontract.png</normaloff>:/images/ucontract.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_admin">
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_reset">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Khoá/Mở khoá tài khoản</string>
        </property>
        <property name="statusTip">
         <string>Khoá/Mở khoá tài khoản này</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <selectedoff>:/images/reset2.png</selectedoff>
         </iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>49</width>
          <height>61</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_lock">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Đặt lại mật khẩu</string>
        </property>
        <property name="statusTip">
         <string>Reset mật khẩu cho tài khoản này</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/select_code-512.png</normaloff>:/images/select_code-512.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_del">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Xoá tài khoản</string>
        </property>
        <property name="statusTip">
         <string>Xoá tài khoản này khỏi hệ thống</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/del3.png</normaloff>:/images/del3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_deall">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Xoá người dùng</string>
        </property>
        <property name="statusTip">
         <string>Xoá người dùng này ra khỏi hệ thống</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/del2.png</normaloff>:/images/del2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_profile">
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_changepass">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Đổi mật khẩu</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/select_code-512.png</normaloff>:/images/select_code-512.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_savepass">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="src.qrc">
          <normaloff>:/images/floppy-disk.png</normaloff>:/images/floppy-disk.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
